src/main/java/com/blackhearth/blockchain/wallet/KeysGenerator.java

```diff
@@ -0,0 +1,32 @@
+package com.blackhearth.blockchain.wallet;
+
+
+import java.security.*;
+
+public class KeysGenerator {
+
+    private KeyPairGenerator keyGen;
+    private PrivateKey privateKey;
+    private PublicKey publicKey;
+
+    public KeysGenerator(int keylength) throws NoSuchAlgorithmException {
+        this.keyGen = KeyPairGenerator.getInstance("RSA");
```

Comment:
Używamy this wtedy gdy jest to potrzebne. W całej tej klasie nie ma potrzeby używania this.